src/assessment_scripts/det_vs_gt.py

```diff
@@ -61,12 +62,15 @@ def file_loader(files):
 
     acc_gdf = gpd.GeoDataFrame() # accumulator
 
-    crs = None # init
+    #crs = None # init
+    crs = epsg_dft
+    
     for _file in files:
         # TODO: check schema  
         tmp_gdf = gpd.read_file(_file) 
-        if crs is None:
-            crs = tmp_gdf.crs
+        
+        if tmp_gdf.crs is None:
+            crs = epsg_dft
```

Comment:
Line 66 already sets `crs = epsg_dft`.

Reply:
In the previous implementation, we used to read the CRS from the first GeoDataFrame to be concatenated. Then, we wanted to make sure that all the subsequent ones had the same CRS. Such a CRS was lost upon pd.concat([...]), as Pandas is not CRS-aware. It was then restored by line 81 `gdf = gdf.set_crs(crs)`.

* What happens with the new implementation?
* How could the previous implementation work with the DFT output, despite the lack of CRS?!

Reply:
About l.66: it was meant for humans "if there is no CRS, the CRS of the project will be set"; but yes, it is otherwise useless. 

What happens with the new implementation ? It has two purposes : 
1. It fixes a CRS for the DFT outputs which have none. 
2. It assures good practice by checking that all layers (detection, GT, sectors) are in the same CRS. This is mandatory for the clip function by example. 

For that reason, I think the fix is improving the previous implementation. Here a cleaner version (with the CRS as argument and no more global variable) : 

`def file_loader(files, crs):
    
    acc_gdf = gpd.GeoDataFrame() # accumulator

    for _file in files:
        # TODO: check schema  
        tmp_gdf = gpd.read_file(_file) 
        
        if tmp_gdf.crs != None:
            if tmp_gdf.crs != crs:
                logger.critical("Input datasets have mismatching CRS. Exiting.")
                sys.exit(1)
        acc_gdf = pd.concat([acc_gdf, tmp_gdf])

    gdf = gpd.GeoDataFrame(acc_gdf)
    gdf = gdf.set_crs(crs)

    return gdf.reset_index(drop=True)`